tools/maint/gen_struct_info.py

```diff
@@ -423,7 +401,9 @@ def main(args):
   extra_cflags = []
 
   if args.wasm64:
-    settings.MEMORY64 = 2
+    # Always use =2 here so that we don't generate binary that actually requires
```

Comment:
```suggestion
    # Always use =2 here so that we don't generate a binary that actually requires
```
A very minor concern is that I imagine we will remove mode 2 some day, when memory64 is so stable we don't need it?

Reply:
Exactly yes.   For now, we need it.   I'm not changing that, by the way.  This is just the status quo.